packages/razorsharp/src/code/blade/components/box/box.ts

```diff
@@ -0,0 +1,144 @@
+import { generateBladeCode } from '../../main';
+import { convertStyleNameToBladeName, isBackgroundColorToken } from '../../utils/color';
+import { component } from '../../utils/component';
+import { bladeImports, mergeImports } from '../../utils/imports';
+import { getPaddingValue, getTokenFromSpacingValue } from '../../utils/spacing';
+import { defaultValues, LAYOUT_MODES } from './constants';
+import { getFlexAlignmentFromAxisAlignment } from './utils';
+import type { TransformFunctionReturnType } from '~/code/types/TransformFunction';
+import type { BladeFrameNode, BladeGroupNode, BladeProps } from '~/code/types/Blade';
+
+export const transformFrameOrGroup = (
+  bladeFrame: BladeFrameNode | BladeGroupNode,
+): TransformFunctionReturnType => {
+  const props: BladeProps = {};
+
+  // TODO groups can have item spacing as well
+  // since item spacing figma property does not exist
+  // for groups, use relative transform matrix to find the
+  // distances between elements in a group
+  if (bladeFrame.type === 'GROUP') {
+    let children: TransformFunctionReturnType = { component: '', imports: {} };
+    if (bladeFrame.children && bladeFrame.children.length > 0) {
+      children = generateBladeCode({
+        bladeNodes: bladeFrame.children,
+      });
+    }
+
+    return {
+      component: component('Box', {
+        props,
+        defaultValues,
+        children: children.component,
+      }),
+      imports: mergeImports(children.imports ?? {}, bladeImports(['Box'])),
+    };
+  }
+
+  // --- Frame specific code below ---
+  // --- Layout mode specific code below ---
+  // TODO set alignItems and justifyContent when items don't fill the entire space
```

Comment:
Hope you're tracking all the todos at a single place as well

Reply:
Not yet. Would you suggest creating issues on the blade repo for this? These todos have been sent to design system team as well but resolution has not been done

Reply:
For now, probably somewhere for your record / slack till there is a resolution